lib/MusicBrainz/Server/Plugin/Diff.pm

```diff
@@ -216,7 +201,6 @@ sub diff_artist_credits {
 
 sub diff {
     my ($self, $old, $new) = @_;
-    my $h = HTML::Tiny->new;
```

Comment:
Not really related. There is already a package-local variable `$h` (created on line 39 of the file) that is used everywhere else except for this function; and there is no need to have a separate instance here.